### PR TITLE
Sync monikers

### PR DESCRIPTION
**Pull request by AlexJerabek**
Instrumenting monikers for sync/async APIs. 

I'm still investigating the bug pictured below, but I think that's something OPS can help fix. We can merge to the Sync branch without fixing that.

image

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[922d144](https://github.com/OfficeDev/office-scripts-docs-reference/commits/922d144cbc5540e3fe77065870c63adb5c0a71ae)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/code-snippets/excel-async-snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/code-snippets/excel-async-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/excel-snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/code-snippets/excel-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.aggregationfunction.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.aggregationfunction.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.application.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.application.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.arrowheadlength.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.arrowheadlength.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.arrowheadstyle.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.arrowheadstyle.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.arrowheadwidth.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.arrowheadwidth.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.autofilltype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.autofilltype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.autofilter.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.autofilter.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.basicdatavalidation.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.basicdatavalidation.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.binding.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.binding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.bindingcollection.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.bindingcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.bindingtype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.bindingtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.borderindex.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.borderindex.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.borderlinestyle.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.borderlinestyle.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.borderweight.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.borderweight.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.builtinstyle.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.builtinstyle.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.calculationmode.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.calculationmode.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.calculationstate.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.calculationstate.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.calculationtype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.calculationtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellborder.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellborder.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellbordercollection.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellbordercollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellproperties.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesborderloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesborderloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfill.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfill.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfillloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfillloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfont.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfont.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfontloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfontloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesformat.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesformatloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesformatloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesprotection.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesprotection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellvalueconditionalformat.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellvalueconditionalformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chart.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartareaformat.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartareaformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxes.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxes.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxis.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxis.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxiscategorytype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxiscategorytype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisdisplayunit.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisdisplayunit.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisformat.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisgroup.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisgroup.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisposition.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisposition.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisscaletype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisscaletype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisticklabelposition.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisticklabelposition.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxistickmark.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxistickmark.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxistimeunit.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxistimeunit.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxistitle.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxistitle.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C21%5Cb7e1f975-26d5-fb7f-b635-d5c5bc9f9be0%5CPullRequest%5C202004212152498580-30%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=f8CuCZWpi7%2FzinPXyvUyxZLFqve8uRBI%2BixmGglgSBU%3D&st=2020-04-21T21%3A50%3A58Z&se=2020-05-22T21%3A55%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Tagging @ElizabethSamuel-MSFT to look over the moniker implementation.

File: generate-docs/scripts/postprocessor.ts [hunk 1]
```diff
@@ -47,9 +47,9 @@ tryCatch(async () => {
     const docsDestination = path.resolve("../../docs/docs-ref-autogen");
 
     console.log(`Deleting old docs at: ${docsDestination}`);
-    // delete everything except the overview file from the /docs folder
+    // delete everything except the 'overview' folder from the /docs folder
```
Comment (ElizabethSamuel-MSFT) [inline generate-docs/scripts/postprocessor.ts:50]:
```suggestion
    // delete everything except the 'overview' folder under the /docs folder
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3e675fb](https://github.com/OfficeDev/office-scripts-docs-reference/commits/3e675fb43b5fe2046fccd40c699bb001d6dddeea)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/code-snippets/excel-async-snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/code-snippets/excel-async-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/excel-snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/code-snippets/excel-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.aggregationfunction.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.aggregationfunction.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.application.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.application.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.arrowheadlength.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.arrowheadlength.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.arrowheadstyle.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.arrowheadstyle.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.arrowheadwidth.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.arrowheadwidth.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.autofilltype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.autofilltype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.autofilter.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.autofilter.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.basicdatavalidation.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.basicdatavalidation.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.binding.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.binding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.bindingcollection.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.bindingcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.bindingtype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.bindingtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.borderindex.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.borderindex.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.borderlinestyle.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.borderlinestyle.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.borderweight.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.borderweight.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.builtinstyle.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.builtinstyle.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.calculationmode.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.calculationmode.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.calculationstate.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.calculationstate.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.calculationtype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.calculationtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellborder.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellborder.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellbordercollection.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellbordercollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellproperties.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesborderloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesborderloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfill.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfill.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfillloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfillloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfont.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfont.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfontloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesfontloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesformat.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesformatloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesformatloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesloadoptions.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesprotection.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellpropertiesprotection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.cellvalueconditionalformat.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.cellvalueconditionalformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chart.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartareaformat.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartareaformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxes.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxes.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxis.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxis.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxiscategorytype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxiscategorytype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisdisplayunit.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisdisplayunit.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisformat.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisgroup.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisgroup.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisposition.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisposition.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisscaletype.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisscaletype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxisticklabelposition.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxisticklabelposition.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxistickmark.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxistickmark.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxistimeunit.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxistimeunit.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel-async/excel/excel.chartaxistitle.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/Sync-Monikers/docs/docs-ref-autogen/excel-async/excel/excel.chartaxistitle.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C21%5Cb7e1f975-26d5-fb7f-b635-d5c5bc9f9be0%5CPullRequest%5C202004212305080196-30%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=LDmWxIjap9q2tnyNMyTDRO%2FuFSa1YJdo%2BY6AbQ1rWv4%3D&st=2020-04-21T23%3A03%3A31Z&se=2020-05-22T23%3A08%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**